tests/test_models.py

```diff
@@ -791,3 +791,49 @@ def test_fileinstance_validation(app, db, dummy_location):
     f = FileInstance.create()
     f.set_uri('x' * 255, 1000, 1000)  # Should not raise
     pytest.raises(ValueError, f.set_uri, 'x' * 256, 1000, 1000)
+
+
+def test_object_version_tags(app, db, dummy_location):
```

Comment:
I can see you just copy/pasted the tests from buckets, but I think the person (i.e. me) who wrote the bucket tests was a bit lazy :-) Would you mind improving them a bit. E.g.:

- Use the ``objects`` test fixture.
- Split into multiple tests (if it makes sense - hopefully with the test fixtures this shouldn't add much extra code.